jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/impl/TaskQueryServiceImpl.java

```diff
@@ -629,6 +629,16 @@ public int getPendingTaskByUserId(String userId) {
     }
 
     @Override
+    public List<TaskSummary> getTasksByGroup(List<String> groupIds) {
+        Map<String, Object> params = new HashMap<String, Object>();
+        params.put("groupIds", groupIds);
+        return (List<TaskSummary>) persistenceContext.queryWithParametersInTransaction(
+                "TasksByPotentialOwnerGroups",
```

Comment:
Possibly another bikeshed, but to be consistent with the naming in the other classes, should we use different name for the query, e.g. `TasksByGroups`?